xml/System.Threading/ReaderWriterLockSlim.xml

```diff
@@ -49,7 +49,10 @@
   
 > [!NOTE]
 >  <xref:System.Threading.ReaderWriterLockSlim> is similar to <xref:System.Threading.ReaderWriterLock>, but it has simplified rules for recursion and for upgrading and downgrading lock state. <xref:System.Threading.ReaderWriterLockSlim> avoids many cases of potential deadlock. In addition, the performance of <xref:System.Threading.ReaderWriterLockSlim> is significantly better than <xref:System.Threading.ReaderWriterLock>. <xref:System.Threading.ReaderWriterLockSlim> is recommended for all new development.  
-  
+
+> [!NOTE]
+>  <xref:System.Threading.ReaderWriterLockSlim> is not thread-abort safe.  Thus, it should not be used in an environment where threads accessing it can be aborted (e.g., ASP.NET.)
```

Comment:
```suggestion
>  <xref:System.Threading.ReaderWriterLockSlim> is not thread-abort-safe. Thus, it should not be used in an environment where threads accessing it can be aborted (e.g., ASP.NET).
```